3.10/installation-windows.md

```diff
@@ -24,6 +25,17 @@ or `certutil -hashfile <filename> sha256` in a terminal.
 Running production environments on Windows is not supported.
 {% endhint %}
 
+Docker
+------
+
+The recommended way of using ArangoDB on Windows is to use the
+[ArangoDB Docker images](https://www.arangodb.com/download-major/docker/){:target="_blank"}
```

Comment:
This seems like an unnecessary indirection. I think we should link directly to Docker Hub.
I believe Vadim would like us to point out the difference between `arangodb` and `arangodb/arangodb` (they are not equal, `arangodb` are the official ones approved by Docker staff, whereas `arangodb/arangodb` are more up-to-date an published by us directly.

Reply:
Added a link to `arangodb/arangodb`Docker images. Other details will be described in DOC-339 on a separate "Install with Docker" page.